aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ systemd:
         - name: 40-etcd-cluster.conf
           contents: |
             [Service]
-            Environment="ETCD_IMAGE_TAG=v3.3.15"
+            Environment="ETCD_IMAGE_TAG=v3.4.1"
```

Comment:
You sure etcd 3.4.1 is supported? https://github.com/kubernetes/kubernetes/issues/82641

Reply:
I think k8s community is in general is slow with adoption of newer base components because they need to be changed in various different locations. And it is a task in itself.

Reply:
FYI, I have been testing etcd v3.4.1 while I was testing my ARM support branch https://github.com/kinvolk/lokomotive-kubernetes/pull/78. (Funny enough, I had to upgrade etcd to v3.4.1 because of container images for etcd v3.3.x did not work with rkt on Flatcar)
So far it was ok for me, no problem at all with etcd v3.4.
Though I'm testing only on Packet.
Actually I'm not sure if any corner case can pop up when upgrading etcd to v3.4.

Reply:
Typhoon seems to be using 3.4.2 (released one week ago)